Pipfile

```diff
@@ -0,0 +1,10 @@
+[[source]]
+url = "https://pypi.python.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+sanpy = {editable = true, path = "."}
+
+[requires]
+python_version = "3.6"
```

Comment:
What does it mean `requires` python 3.6  in this case ? If I have only 3.7 or 3.8
will it be ok. Generally we need something like >= 3.6 (or >= 3.0 if we want to support all python 3)

Reply:
The file will be used only for development, `pipenv` will create a virtualenv and install the required python for you. The idea is to have the minimal supported version locked so everyone is using the same one and not accidentally introduce code that will not work on that version.

Reply:
Here is a nice article about pipenv: http://totempaal.tj/2018/07/09/python-dependency-management.html

Reply:
@kalkov thanks for the link